libpolys/polys/monomials/p_polys.cc

```diff
@@ -4529,8 +4529,8 @@ static inline unsigned long GetBitFields(const long e,
 // if (ev1 & ~ev2) then exp1 does not divide exp2
 unsigned long p_GetShortExpVector(const poly p, const ring r)
 {
-  assume(p != NULL);
```

Comment:
p_GetShortExpVector is one of the routine that must not be called with p==NULL.
Returning 0 is a reallay bad idea as 0 stand here for p being a NON-ZERO- constant polynomial.
If this routine is called with p==NULLm the caller is wrong!